extension-manifest-v2/package.json

```diff
@@ -48,8 +48,9 @@
   },
   "homepage": "https://github.com/ghostery/ghostery-extension#readme",
   "dependencies": {
-    "@ghostery/ui": "../ui",
     "@cliqz/url-parser": "^1.1.4",
+    "@ghostery/libs": "file:../libs",
```

Comment:
Is there a reason why "libs" and "ui" use different syntax ("file:../libs" vs "../ui")? Should it also be "file:../ui"? (I'm not familiar with local paths in package.json)

Reply:
there is no difference AFAICT